cmd/zetae2e/local/v2.go

```diff
@@ -20,7 +20,11 @@ func startV2Tests(eg *errgroup.Group, conf config.Config, deployerRunner *runner
 		e2etests.TestV2ETHDepositAndCallName,
 		e2etests.TestV2ETHWithdrawName,
 		e2etests.TestV2ETHWithdrawAndCallName,
+		e2etests.TestV2ETHWithdrawAndAuthenticatedCallName,
+		e2etests.TestV2ETHWithdrawAndAuthenticatedCallThroughContractName,
```

Comment:
Might be refactored in a further PR but I think we set these as the standard withdraw test and rename the current withdraw tests into `WithdrawArbitraryCall`